data-catalog-entry-schema.json

```diff
@@ -0,0 +1,263 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "MPI AB EAS Data Catalog Schema",
+  "type": "object",
+  "properties": {
+    "data_catalog_entry_id": {
+      "title": "Data Catalog Entry Id",
+      "description": "This is unique identifier for the entry you data catalog entry you want to archive. For example one could use `coiba_capuchins_july_2021` or `kob_apple_data_2021`.",
+      "type": "string"
+    },
+    "data_catalog_entry_details": {
+      "title": "Data Catalog Entry Details",
+      "description": "This is particular info about the entry.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "project_name": {
+            "title": "Project Name",
+            "description": "This is the name of your server folder if it is a long term project. It is a drop down menu which needs to get updated as projects are added.",
+            "type": "string",
+            "enum": [
```

Comment:
In this version "project_name" must be on this list.  Should it there be enforcement there or should this just be a list of examples?